cmd/orb.go

```diff
@@ -366,17 +369,38 @@ func logOrbs(orbCollection *api.OrbsForListing) error {
 	return nil
 }
 
+var validSortFlag = map[string]bool{
+	"builds":   true,
+	"projects": true,
+	"orgs":     true}
+
+func validateSortFlag(sort string) error {
+	if _, valid := validSortFlag[sort]; valid {
+		return nil
+	}
+	// TODO(zzak): we could probably reuse the map above to print the valid values
```

Comment:
There's a few approaches in this [SO thread](https://stackoverflow.com/a/36387643/640517), but I liked this one for its simplicity:

`keys := reflect.ValueOf(abc).MapKeys()`

Reply:
Let's fix this in a follow up PR :bow: